graylog2-web-interface/src/stores/indexers/IndexerOverviewStore.js

```diff
@@ -0,0 +1,25 @@
+import Reflux from 'reflux';
+
+import URLUtils from 'util/URLUtils';
+import ApiRoutes from 'routing/ApiRoutes';
+import fetch from 'logic/rest/FetchProvider';
+
+import ActionsProvider from 'injection/ActionsProvider';
+const IndexerOverviewActions = ActionsProvider.getActions('IndexerOverview');
+
+const IndexerOverviewStore = Reflux.createStore({
+  listenables: [IndexerOverviewActions],
+  list() {
+    const url = URLUtils.qualifyUrl(ApiRoutes.IndexerOverviewApiResource.list().url);
+    const promise = fetch('GET', url);
+    promise.then((response) => {
+      this.trigger(response);
```

Comment:
I don't really like this, as the response is returning several keys that will go into a component's state (at least if it's connecting to the store), and there's no obvious way of knowing which keys they are unless you look into the response. Can we make the keys explicit here?